src/zjs_sensor.c

```diff
@@ -406,7 +403,22 @@ static void zjs_sensor_onstop_c_callback(void *h, void *argv)
     send.data.sensor.channel = handle->channel;
 
     ZVAL reading_val = jerry_create_null();
-    zjs_set_property(obj, "reading", reading_val);
+    jerry_value_t null_val = jerry_create_null();
+    switch(handle->channel) {
+    case SENSOR_CHAN_ACCEL_XYZ:
+    case SENSOR_CHAN_GYRO_XYZ: ;
```

Comment:
Oh maybe you couldn't declare a new variable inside? But yeah you could just do that above the switch, or possibly before the cases in the switch. Or open a new {} context.

Anyway there are still two more of these in the file I think. Search for ": ;" - they can be removed.